src/components/Tutorial.tsx

```diff
@@ -0,0 +1,29 @@
+import Link from 'next/link'
+
+export default function Tutorial() {
+  return (
+    <div
+      id="Tutorial"
+      className="mb-5 mt-10 flex flex-col items-center justify-center"
+    >
+      <h1 className="mb-4 text-center text-4xl font-extrabold text-primary">
+        How it works:
+      </h1>
+      <ol className="list-inside list-decimal text-xl">
+        <li>
+          Create a new FindingATime event at &apos;Create Event&apos; using the
```

Comment:
There's a few of these in the code but do you mind changing this to "Finding A Time" from "FindingATime".
It's an attempt to make the phrase the entity and the action since appended together makes it look a bit clunky.

Reply:
I think the create user changes in here are because this branch is branched off the userLocalStorage branch. As for the name, should I change it to "Finding A Time" everywhere or leave it in the header as the app name as "FindingATime"?

Reply:
Grab the change in a follow up change you make. Let's get this in.